yaml/_dumper.ts

```diff
@@ -653,8 +653,7 @@ function writeBlockSequence(
 function writeFlowMapping(
   state: DumperState,
   level: number,
-  // deno-lint-ignore no-explicit-any
-  object: any,
+  object: Record<PropertyKey, unknown>,
```

Comment:
Ditto. This argument can be removed and we instead use `this.dump` within the method.